ipaserver/plugins/topology.py

```diff
@@ -82,7 +82,12 @@
 
 
 def validate_domain_level(api):
-    current = int(api.Command.domainlevel_get()['result'])
+    current = DOMAIN_LEVEL_0
+    try:
+        current = int(api.Command.domainlevel_get()['result'])
+    except errors.NotFound:
+        pass
```

Comment:
I don't like the default + override style and rather prefer retrieve / except default style. It's also one line shorter:
```
try:
    current = ...
except ...:
    current = DOMAIN_LEVEL_1
```